storageservice/modules/files/Module.md

```diff
@@ -1,89 +1,39 @@
+## Overview
+This module allows you to access the Azure Storage File REST API through Ballerina.
+It provides you the capability to execute operations like getFileList, getDirectoryList, createDirectory, directUpload and getFile etc using the FileClient. It also allows you to execute management operations such as createShare and deleteShare etc using the ManagementClient.
```

Comment:
Let's add a comprehensive description of the module.

Reply:
Done